modules/cms-kit/src/Volo.CmsKit.Common.Application/Volo/CmsKit/Menus/MenuUpdatedHandler.cs

```diff
@@ -0,0 +1,23 @@
+using System.Threading.Tasks;
+using Volo.Abp.Caching;
+using Volo.Abp.DependencyInjection;
+using Volo.Abp.Domain.Entities.Events;
+using Volo.Abp.EventBus;
+
+namespace Volo.CmsKit.Menus
+{
+    public class MenuUpdatedHandler :  ILocalEventHandler<EntityUpdatedEventData<Menu>>, ITransientDependency
```

Comment:
We should also handle `EntityDeletedEventData<Menu>` to remove the deleted object from the cache.